blog_posts/settings.py

```diff
@@ -37,9 +38,13 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'rest_framework',
+    'blogapi',
+    'corsheaders'
 ]
 
 MIDDLEWARE = [
+    'corsheaders.middleware.CorsMiddleware',
```

Comment:
this can't be right

![image](https://user-images.githubusercontent.com/20076368/175108536-5da8f70f-2774-4db5-afc8-a91b2dcb80ed.png)

https://github.com/adamchainz/django-cors-headers